### PR TITLE
feat: add quickstart models and sorting

### PR DESCRIPTION
**Pull request by TheLLspectre**
Adding quickstart tab and sorting models.
Adding documentation.
Fix some variable naming.

### CODE REVIEW

File: package/Editor/Models/Models.cs [hunk 1]
```diff
@@ -12,12 +12,15 @@ public class Models : EditorWindow
 
         #region Public Fields
 
+        public static List<ModelData> modelsQuickStart = new();
```
Comment (Morgan-6Freedom) [inline package/Editor/Models/Models.cs:15]: please add some comments to your newly added Lists

File: package/Editor/Models/Models.cs [hunk 2]
```diff
@@ -12,12 +12,15 @@ public class Models : EditorWindow
...
         public static List<ModelData> modelsPrivate = new();
         public static List<ModelData> modelsPublic = new();
 
+        public static List<TexturePair> texturesQuickStart = new();
```
Comment (Morgan-6Freedom) [inline package/Editor/Models/Models.cs:19]: please add some comments to your newly added Lists

File: package/Editor/Models/Models.cs [hunk 3]
```diff
@@ -12,12 +12,15 @@ public class Models : EditorWindow
...
         public static List<TexturePair> texturesPrivate = new();
         public static List<TexturePair> texturesPublic = new();
 
+        public static string privacyQuickStart = "quickstart";
```
Comment (Morgan-6Freedom) [inline package/Editor/Models/Models.cs:23]: please add some comments to your newly added Lists

File: package/Editor/Models/Models.cs [hunk 4]
```diff
@@ -209,19 +319,75 @@ private static void FetchAllPublicTextures()
...
-        private static async Task FetchAllPrivateModels()
+        /// <summary>
+        /// Processing to get all quickstart models
+        /// </summary>
+        /// <returns></returns>
+        private static async Task FetchAllQuickStartModels()
         {
             if (!isProcessing)
             {
+                while (true)
```
Comment (Morgan-6Freedom) [inline package/Editor/Models/Models.cs:330]: Why is there a while(true) here ? seems weird / useless

File: package/Editor/Models/ModelsUI.cs [hunk 5]
```diff
@@ -32,8 +32,14 @@ public class ModelsUI
...
             pageModels.Clear();
-
-            selectedTab = (Models.IsPrivateTab()) ? 0 : 1;
+            if (IsQuickStartTab())
+            {
+                selectedTab = 0;
+            }
+            else
+            {
+                selectedTab = (IsPrivateTab()) ? 1 : 2;
```
Comment (Morgan-6Freedom) [inline package/Editor/Models/ModelsUI.cs:41]: selectedTab seems redundant with privacySettings in Models.cs

**Review by Morgan-6Freedom — CHANGES_REQUESTED** [review]

File: package/Editor/Models/Models.cs (hunk 4 above)
Comment (TheLLspectre) [reply package/Editor/Models/Models.cs:363]: So without this while, is fetching only one page of model.
But I'm agree, and may be we can use thoses lines to:
```
 PaginationTokenPublic = "";
 Debug.Log("no next page to fetch.");
```
To stop the process. I'll check this

Comment (TheLLspectre) [reply package/Editor/Models/Models.cs]: Fix it !

With this fix we also fix another bug, which it was, sometines on redraw or reopening, first models or the first page were in double.

**Comment by qvaleroo** [comment]
The model screen is not really usable :/ 
When I switch tabs, they disappear. If I go on page 2 of a tab and then change the tab, and go back, the bottom button is not in sync with what is displayed.
Really long to load, and the display is sporadic.
Can't we do better 🙏 ?
https://github.com/scenario-labs/Scenario-Unity/assets/95273550/1f0dd49c-504d-4ae9-a71d-2051a35abc6b

**Comment by Morgan-6Freedom** [comment]
@qvaleroo can you validate that the last push fix your last comment ?

**Comment by qvaleroo** [comment]
The last fix forces the quick start to be opened first, but the tab still disappears, and the display of models is still chaotic. But it is already the case in production. 
This should be addressed really soon I think, but we can merge this one to address the Quickstart tab

**Comment by Morgan-6Freedom** [comment]
@qvaleroo I merge this one, can you open a bug for the disappearance ?

**Review by Morgan-6Freedom — APPROVED** [review]

**Comment by qvaleroo** [comment]
https://github.com/scenario-labs/Scenario-Unity/issues/157

**Pull request merged**